azkaban-web-server/src/test/java/azkaban/flowtrigger/TriggerInstanceProcessorTest.java

```diff
@@ -107,6 +108,7 @@ public void setUp() throws Exception {
     final CommonMetrics commonMetrics = new CommonMetrics(new MetricsManager(new MetricRegistry()));
     this.emailer = Mockito.spy(new Emailer(EmailerTest.createMailProperties(), commonMetrics,
         this.messageCreator));
+
```

Comment:
extra line

Reply:
thanks, this is intentional left for separation.